src/lib.rs

```diff
@@ -65,6 +67,7 @@ use crate::iterators::{
     NodesCountMapping, PathLengthMapping, PathMapping, Pos2DMapping,
     WeightedEdgeList,
 };
+use crate::steiner_tree::*;
```

Comment:
```suggestion
use steiner_tree::*;
```

Module is declared on the same file, we don't need the `crate::` part